CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Change Log
```

Comment:
Not sure we need a changelog for the very first release.

Reply:
I don't think it's a problem to have one that exists.  We can decide when in the development cycle it should start being used (e.g., pysatModels).